client/app/mod/competitors/PersonForm.tsx

```diff
@@ -151,6 +151,7 @@ const PersonForm = ({
       onCancel={onCancel}
     >
       {personUnderEdit && <CreatorDetails creator={personUnderEdit.creator} />}
+      {personUnderEdit && <p>ID: {personUnderEdit.personId}</p>}
```

Comment:
Can you make this say "CC ID:" to make it consistent with the column in the table?

Reply:
```suggestion
      {personUnderEdit && <p>CC ID: {personUnderEdit.personId}</p>}
```